qdrant_client/embed/builtin_embedder.py

```diff
@@ -0,0 +1,94 @@
+from typing import Optional, Any
+
+from qdrant_client.http import models
+from qdrant_client.embed.models import NumericVector
+
+
+class BuiltinEmbedder:
```

Comment:
Maybe add a docstring clarifying that this is *server* builtin, not client builtin. (Maybe it's just me that got confused by this.)